plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -226,17 +218,20 @@ SecRule REQUEST_FILENAME "@contains /apps/text/session/create" \
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
 # FP while trying to open text files in file manager
-SecRule REQUEST_FILENAME "@contains /remote.php/dav/comments" \
+# Adding/editing comments to files
+# The content of comments could be anything
+SecRule REQUEST_FILENAME "@rx /remote.php/dav/comments/files/[0-9]+(?:/[0-9]+)?$" \
```

Comment:
```suggestion
SecRule REQUEST_FILENAME "@rx /remote\.php/dav/comments/files(?:/[0-9]+)+$" \
```

Reply:
That's intentional.
the rule is meant to match both:
```
/remote.php/dav/comments/files/123/45
/remote.php/dav/comments/files/123
```
I'll fix the unescaped ``.`` character.

Reply:
My version matches both. It's equivalent to what you wrote initially, just shorter.

Reply:
you're right, I just tested with regexr. Fixed.